src/app.html

```diff
@@ -6,7 +6,7 @@
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <link
       rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0..1,0" />
```

Comment:
We are only using this size icon, and this significantly reduces the size of the font